pages/linux/unshare.md

```diff
@@ -0,0 +1,12 @@
+# unshare
+
+> Execute a command in new user-defined namespaces.
+> More information: <https://www.kernel.org/doc/html/latest/userspace-api/unshare.html>.
+
+- Execute a command without sharing access to connected networks:
```

Comment:
```suggestion
- Execute the specified command without sharing access to connected networks:
```

Reply:
I'm actually partial to this one as well, anyone else care to drop an opinion?

Reply:
a works better.